conf/muchos.props.example

```diff
@@ -30,7 +30,7 @@ install_dir = %(user_home)s/install
 # from your machine. Hostname can be chosen from "nodes" section below.
 proxy_hostname = leader1
 # If set, a SOCKS proxy will be created on the specified port when connecting to proxy using 'muchos ssh <cluster>'
-#proxy_socks_port = 38585
+proxy_socks_port = 38585
```

Comment:
Is this change required due to the default OS changing?

Reply:
No. I just uncommented it while testing and left it that way because it's super useful. I didn't see a reason why it was commented out by default. I can put it back, but I'm not sure there's a good reason to leave it off by default.